### PR TITLE
Add base for natural language processing

### PR DESCRIPTION
**Pull request by Nanosync**
#27

### CODE REVIEW

**Comment by austinsantoso** [comment]
Looks good to merge

**Review by jonathantjendana — APPROVED** [review]
Looks good to merge!

**Review by austinsantoso — APPROVED** [review]
lgtm

File: src/test/java/seedu/address/commons/util/DateUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Tests DateUtil.
+ */
+class DateUtilTest {
+
+    @Test
+    public void parseDate_validDate() throws ParseException {
```
Comment (choongyx) [inline src/test/java/seedu/address/commons/util/DateUtilTest.java:19]: The convention for test names is `methodName_testScenario_expectedResult`. expectedResult seems to be missing.

**Pull request merged**